docs/admin/guides/link_service.md

```diff
@@ -0,0 +1,29 @@
+# Link services to local ports
+
```

Comment:
Please add the requirements:
1. User needs to have an account on server2.
2. SSH server must be running on server2